snap/snapcraft.yaml

```diff
@@ -150,7 +150,7 @@ parts:
 
       mkdir -p .gopath/src/github.com/edgexfoundry
       export GOPATH="$PWD/.gopath"
-      export PATH="$GOPATH/bin:$PATH:$SNAPDIR/parts/glide/install"
+      export PATH="$GOPATH/bin:$PATH:$SNAPCRAFT_STAGE"
```

Comment:
I don't think this is correct as is with the `glide` part because the glide part is installed into `$SNAPCRAFT_PART_INSTALL` which is something like what it was before `$SNAPDIR/parts/glide/install`, and not `$SNAPCRAFT_STAGE` which is something like `$SNAPDIR/stage`. I would be fine with a change to the glide part that makes glide installed into `$SNAPCRAFT_STAGE` and then was filtered out from the final snap in the glide part with prime: 
```
  prime:
    - -*
```